build_docs.pl

```diff
@@ -55,7 +55,7 @@ BEGIN
     $Opts,    #
     'all', 'push', 'target_repo=s', 'reference=s', 'rebuild', 'no_fetch', #
     'single',  'pdf',     'doc=s',           'out=s',  'toc', 'chunk=i', 'suppress_migration_warnings',
-    'open',    'skiplinkcheck', 'linkcheckonly', 'staging', 'procs=i',         'user=s', 'lang=s',
+    'open',    'skiplinkcheck', 'linkcheckonly', 'procs=i',         'user=s', 'lang=s',
     'lenient', 'verbose', 'reload_template', 'resource=s@', 'asciidoctor', 'in_standard_docker',
```

Comment:
Not your fault, but you would you mind doing something about this whitespace when you're next in here?

Reply:
How about I open a PR after this that sorts them? Some of them are "for `--all`" and some are "for `--doc`" and some of them are "common" so I shouldn't sort them blindly. But I think a followup PR is best here.

Reply:
Thanks! Yes. I really meant "if you are bored sometime in the future". <3